src/js/components/Drop/Drop.js

```diff
@@ -16,35 +15,18 @@ class Drop extends Component {
     },
   }
 
-  componentDidMount() {
-    this.dropContainer = getNewContainer();
-    this.renderDrop();
-  }
-
-  componentDidUpdate() {
-    this.renderDrop();
-  }
+  dropContainer = getNewContainer();
 
   componentWillUnmount() {
-    if (this.dropContainer) {
-      unmountComponentAtNode(this.dropContainer);
-      document.body.removeChild(this.dropContainer);
-    }
+    document.body.removeChild(this.dropContainer);
```

Comment:
Since we have getNewContainer(), I'm thinking we should have a corresponding function to remove it. That way, all of the `document.body` stuff is in the same place. What about naming them: `addDOMContainer()` and `removeDOMContainer()`?